test/basic_test.py

```diff
@@ -127,6 +129,16 @@ def test_api_clients(self):
                 print('Exception when calling Ticketing API: %s' % e)
                 raise e
 
+        # Test ENUM serialization
+        ticketing_category = CategoriesEnum("ticketing")
+        assert ticketing_category.value == "ticketing"
+
+        # List LinkedAccounts test enum serialization and deserialization
+        api_client = ApiClient(ticketing_configuration)
+        linked_account_api_instance = LinkedAccountsApi(api_client)
+        api_response = linked_account_api_instance.linked_accounts_list(category=ticketing_category.value).results
+        assert api_response[0].category == CategoriesEnum("ticketing").value
```

Comment:
Test retrieving `CategoriesEnum` from response

Reply:
For a non-category enum, can you add a test for deserializing a non-standard value? Like, we wont' return anything different here for a long while outside of the 5 standard categories, but for ats url.type or hris martial_status we sometimes return a value outside of the common ones. Just want to make sure that still works.